pwnlib/term/unix_termcap.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import division
+
+__all__ = ['get']
+import curses
+import os
+import sys
+
+cache = None
+
+def get(cap, *args, **kwargs):
+    default = kwargs.pop('default', '')
+
+    if 'PWNLIB_NOTERM' in os.environ:
+        return ''
+
+    # Hack for readthedocs.org
+    if 'READTHEDOCS' in os.environ:
+        return ''
+
+    if kwargs != {}:
+        raise TypeError("get(): No such argument %r" % kwargs.popitem()[0])
+
+    if cache == None:
```

Comment:
This is a good chance to fix singleton comparison:
```suggestion
    if cache is None:
```